src/pages/resultsView/enrichments/MRNAEnrichmentsTab.tsx

```diff
@@ -35,6 +36,21 @@ export default class MRNAEnrichmentsTab extends React.Component<IMRNAEnrichments
                         selectedValue={this.props.store.selectedEnrichmentMRNAProfile.molecularProfileId}
                         molecularProfileIdToProfiledSampleCount={this.props.store.molecularProfileIdToProfiledSampleCount}/>
                     <ExpressionEnrichmentContainer data={this.props.store.mRNAEnrichmentData.result!}
+                    groups={[
+                        {
+                            name: "Altered group",
+                            description: "samples that have alterations in the query gene(s).",
+                            nameOfEnrichmentDirection: "Over-expressed",
+                            count: this.props.store.alteredSampleKeys.result!.length,
+                            color: this.uniqueColorGetter()
```

Comment:
get these colors once up top as private members because otherwise every time the component reloads they'll be different colors. same for protein